src/providers/bedrock.ts

```diff
@@ -403,9 +403,18 @@ export const getLlamaModelHandler = (version: LlamaVersion) => {
       // Check for standard usage format
       if (responseJson?.usage) {
         return {
-          prompt: responseJson.usage.prompt_tokens,
-          completion: responseJson.usage.completion_tokens,
-          total: responseJson.usage.total_tokens,
+          prompt:
+            typeof responseJson.usage.prompt_tokens === 'string'
```

Comment:
nit: A `coerceStrToNum(s: string): number` utility function would reduce a significant amount of boilerplate in this PR.